ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -455,6 +455,12 @@ def _full_backup_restore_with_vault(self, reinstall=False):
                                      '--uninstall',
                                      '-U'])
 
+            # check that no error message in uninstall log for KRA instance
+            ipaserver_uninstall_log = self.master.get_file_contents(
```

Comment:
Instead of transferring the file from master to controller, the test can directly check the command output:
```
cmd = self.master.run_command(...)
assert "failed to uninstall KRA" not in cmd.stdout_text
```

Reply:
Yep it makes sense as it is seen on console and logfile both.